public/index.html

```diff
@@ -1,11 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>Phantom</title>
-  </head>
-  <body>
-    <div id="root"></div>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="icon" type="image/png"
+    href="https://res.cloudinary.com/feyton/image/upload/v1644861999/Codebandits/codebandits_favicon_ahnxce.png">
+  <title>Phantom</title>
+  <meta name="author" content="Code bandits">
+  <meta property="og:image"
+    content="https://res.cloudinary.com/feyton/image/upload/v1644861999/Codebandits/codebandits_favicon_ahnxce.png" />
+  <meta property="og:image:alt" content="Phantom Team Project" />
+</head>
+
+<body>
+  <div id="root"></div>
+</body>
+
+</html>
```

Comment:
Some space after the last line of the file is needed.

Reply:
Fixed